charts/coredns/Chart.yaml

```diff
@@ -19,6 +19,8 @@ engine: gotpl
 type: application
 annotations:
   artifacthub.io/changes: |
+    - kind: added
+      description: Added podSecurityContext
```

Comment:
Please remove all the other entries in the changelog and keep only the one relevant to this PR